hubbard/plot/spectrum.py

```diff
@@ -167,18 +167,18 @@ class DOS_distribution(GeometryPlot):
     ----------
     HH: HubbardHamiltonian
         mean-field Hubbard Hamiltonian
-    PDOS: numpy.ndarray
+    LDOS: numpy.ndarray
```

Comment:
I think we may have misunderstood each other. This was correct. Your latest commit can be reverted as far as I can tell.